js/models/Address.js

```diff
@@ -29,6 +29,10 @@ class Address
     toString() {
         return this._text;
     }
+
+    isValid() {
+        return this._text && this._lat && this._lng;
```

Comment:
``return Boolean( this._text && this._lat && this._lng )``

Reply:
``return Boolean( this._text && (typeof this._lat == "number") && (typeof this._lng == "number") );``